sensing/pointcloud_preprocessor/include/pointcloud_preprocessor/concatenate_data/concatenate_and_time_sync_nodelet.hpp

```diff
@@ -85,6 +85,24 @@
 #include <tf2_ros/buffer.h>
 #include <tf2_ros/transform_listener.h>
 
+// replace topic name postfix
+inline std::string replace_topic_name_postfix(
+  const std::string & original_topic_name, const std::string & postfix)
+{
+  // separate the topic name by '/' and replace the last element with the new postfix
+  size_t pos = original_topic_name.find_last_of("/");
+  if (pos == std::string::npos) {
+    // not found '/': this is not a namespaced topic
+    std::cerr
```

Comment:
@YoshiRi 
Is it possible to check at the caller of the function, since with this method of debug printing, it's unclear which node is outputting? 
If it's difficult to implement, it's fine to leave it as is, but my opinion is that it would be preferable for being able to understand the significance of the output to the terminal (such as ERROR, INFO, etc.).

Reply:
@kyoichi-sugahara 
I moved this functions to class member.
We can get following warning:

> [component_container_mt-1] [WARN 1709309799.455278788] [sensing.lidar.concatenate_data]: The topic name /sensing/lidar/top/pointcloud have the same postfix with synchronized pointcloud. We use the postfix to the end of the topic name. (replaceSyncTopicNamePostfix() at /home/yoshiri/autoware_latest/src/universe/autoware.universe/sensing/pointcloud_preprocessor/src/concatenate_data/concatenate_and_time_sync_nodelet.cpp:255)